knowledge-base/html5-viewer-export-operations-fail-in-chrome-and-firefox-with-err_connection_reset.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Export operations fail in Chrome and Firefox with ERR_CONNECTION_RESET
+description: Export operations fail in Chrome and Firefox with ERR_CONNECTION_RESET.
+type: how-to
+page_title: Export operations fail in Chrome and Firefox with ERR_CONNECTION_RESET
+slug: html5-viewer-export-operations-fail-in-chrome-and-firefox-with-err_connection_reset
+res_type: kb
+---
+
+## Environment
+<table>
+	<tbody>
+		<tr>
+			<td>Product</td>
+			<td>Progress® Telerik® Reporting HTML5 Report Viewer</td>
+		</tr>
+	</tbody>
+</table>
+
+## Description
+
+Export operations initiated through the HTML5 Viewer result in **ERR\_CONNECTION\_RESET** error in **Firefox** and **Chrome** browsers when **Telerik UI for ASP.NET AJAX RadCompression** is *enabled*.  
+  
+## Solution  
+
+The **RadCompression** must be disabled for request to the Reporting REST service ([WebAPI based](../telerik-reporting-rest-conception)) in order documents bytes to be received by the client(viewer).  
```

Comment:
Avoid passive voice! Reword this to: "Disable the **RadCompression** to the Reporting REST service to receive documents bytes in the client (Viewer)"